rebasehelper/logger.py

```diff
@@ -73,11 +78,29 @@ def add_file_handler(logger_object, path, formatter_object=None, level=None):
         logger_object.addHandler(file_handler)
 
 
+class ColorizingStreamHandler(logging.StreamHandler):
+    level_map = {
+        logging.DEBUG: {'fg': 'brightblack', 'bg': 'default', 'style': None},
+        logging.INFO: {'fg': 'default', 'bg': 'default', 'style': None},
+        logging.WARNING: {'fg': 'yellow', 'bg': 'default', 'style': None},
+        logging.ERROR: {'fg': 'red', 'bg': 'default', 'style': 'bold'},
+        logging.CRITICAL: {'fg': 'white', 'bg': 'red', 'style': 'bold'},
+    }
+
+    def emit(self, record):
+        try:
+            message = self.format(record)
+            rebasehelper.utils.ConsoleHelper.cprint(message, **self.level_map.get(record.levelno, None))
```

Comment:
No need for `None` in `get()` method, as that's the default. But you should rather use `{}` instead, because `**None` leads to `TypeError`.